test/cli/same-loc/same-loc.out

```diff
@@ -10,22 +10,6 @@ test/cli/same-loc/same-loc.rb:4: Unable to resolve constant `ODM` https://srb.he
      4 |  const :foo, T::Array[Foo::Baz]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-test/cli/same-loc/same-loc.rb:4: Unable to resolve constant `ODM` https://srb.help/5002
-     4 |  const :foo, T::Array[Foo::Baz]
-          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-test/cli/same-loc/same-loc.rb:4: Unable to resolve constant `Foo` https://srb.help/5002
-     4 |  const :foo, T::Array[Foo::Baz]
-                               ^^^
-
-test/cli/same-loc/same-loc.rb:4: Unable to resolve constant `Foo` https://srb.help/5002
-     4 |  const :foo, T::Array[Foo::Baz]
-                               ^^^
-
-test/cli/same-loc/same-loc.rb:4: Unable to resolve constant `Foo` https://srb.help/5002
-     4 |  const :foo, T::Array[Foo::Baz]
-                               ^^^
-
```

Comment:
This test runs with --no-stdlib. I think at this point, whatever it was trying to test originally it's not testing. I think we might want to just delete the test.